.github/workflows/ci.yaml

```diff
@@ -3,7 +3,7 @@ name: ci
 on:
   push:
     branches:
-      - "*"
+      - '**'
```

Comment:
TIL - `*` will run on all branches not containing `/`, and `**` will run on all branches
https://stackoverflow.com/a/57903434/886537